### ISSUE TITLE
Module not found: Can't resolve './Async.js'

### ISSUE THREAD

**Issue by adhenrique** [ghengeveld/react-async#59]
Hi guys.
I'm trying to use the [helpers components](https://github.com/ghengeveld/react-async#with-helper-components) template, but I get an error:
`Module not found: Can't resolve './Async.js'`

Is this version broken?
I used `npx create-react-app`.

My `app.js` code:
```
import React from 'react';
import axios from 'axios';
import Async from "react-async";

const loadRepos = async () => {
    await axios
        .get('https://api.github.com/users/adhenrique/repos')
        .then(res => (res.ok ? res : Promise.reject(res)))
        .then(res => res.json());
};

const ReactAsync = () => {
    return (
        <Async promiseFn={loadRepos}>
            <Async.Loading>Loading...</Async.Loading>
            <Async.Fulfilled>
                ok :D
            </Async.Fulfilled>
            <Async.Rejected>{error => `Something went wrong: ${error.message}`}</Async.Rejected>
        </Async>
    )
};

export default ReactAsync;
```

React: 16.8.6
React-router: ^ 5.0.1
windows 10 pro
node: 10.15.0
npm: 6.4.1

**Comment by ghengeveld** [comment]
Hmm that certainly doesn't look good. With the latest version (v7.0.0) we migrated to a new project setup, it's possible that this has broken something somewhere. Can you try `npm install react-async@6.2.2`? If that works there's definitely something wrong with the latest release. I'll investigate.

**Comment by ghengeveld** [comment]
Ok so it turns out pack is not picking up the Babel config properly anymore, causing an invalid build. Expect a fix tomorrow.

**Comment by ghengeveld** [comment]
Fixed in [v7.0.1](https://github.com/ghengeveld/react-async/releases/tag/react-async%407.0.1). Thanks for reporting!

**Comment by adhenrique** [comment]
Thanks to update @ghengeveld
